docs/metrics.asciidoc

```diff
@@ -60,3 +60,71 @@ This value is normalized by the number of CPU cores and it ranges from 0 to 100%
 
 The Resident Set Size,
 the amount of memory the process occupies in main memory (RAM).
+
+[float]
+[[metric-nodejs.active_handles]]
+=== `nodejs.active_handles`
```

Comment:
There's still some contention around units in names, but there's also these two. In ECS, there's the following guideline:

> General to specific. Organise the nesting of field sets from general to specific, to allow grouping fields into objects with a prefix like host.*.

Following that guideline, I think we should call these `nodejs.handles.active` and `nodejs.requests.active`. What do you think?

Reply:
Good point. I actually like `nodejs.handles.active` better as it opens up for future handle metrics to be added under `nodejs.handles.*` leter 😃